client/users.go

```diff
@@ -76,7 +76,16 @@ func (s *azureClient) ListAzureADUsers(ctx context.Context, filter string, searc
 			errResult = azure.UserResult{}
 			nextLink  string
 		)
-
+		selectCols := []string{
+			"id",
+			"displayName",
+			"jobTitle",
+			"lastPasswordChangeDateTime",
+			"mail",
+			"accountEnabled",
+			"onPremisesSecurityIdentifier",
+			"onPremisesSyncEnabled",
+		}
 		if users, err := s.GetAzureADUsers(ctx, filter, search, orderBy, selectCols, 999, false); err != nil {
```

Comment:
According to https://learn.microsoft.com/en-us/graph/aad-advanced-queries?tabs=http#user-properties 
```
However, these advanced query capabilities are not available by default but, the requestor must also set the ConsistencyLevel header to eventual and, with the exception of $search, use the $count query parameter. The ConsistencyLevel header and $count are referred to as advanced query parameters.
```

It looks like this `$count` parameter should be set to `true`?

Reply:
I take this back - if we are using `select` we don't need to set `count` to true. Only if we use `filter` or `search`.